packages/app/src/utils/test/TestRoot.tsx

```diff
@@ -33,17 +33,21 @@ const TestRoot = ({ children }: PropsWithChildren<{}>) => {
       },
     }),
   );
+  const value = useMemo(
```

Comment:
That's good in general. Just to understand this: Did you notice a performance issue? In the Test case??

Reply:
I added `useMemo` because my local sonarLint reminded me there would be a performance issue. Without this change it doesn't pass the lint check.